### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by rxaviers** [rxaviers/cldr-data-bower#4]
On https://github.com/jquery/globalize/issues/459#issuecomment-124868488 @johnnyreilly has made a couple of questions which basically shows we need improvements for:
- [ ] Emphasize that the postinstall hook (`.bowerrc`) is required before one runs `bower install`.
- [ ] Clarify Usage.

**Comment by johnnyreilly** [comment]
One part that I still don't understand at all is this section:

> On the bower.json of your application, list the i18n dependencies as you'd normally do. For example, let's say they are FooNumberFormat and BarDateFormat.
```
"dependencies": {
  "foo-number-format": "1.2.3",
  "bar-date-format": "4.5.6"
}
```

I have no idea at all what this means. I get that `FooNumberFormat` and `BarDateFormat` aren't real things but I don't know what they're supposed to represent in this context at all.  I think an example would be very helpful here..

**Comment by rxaviers** [comment]
I agree it's very confusing. The only thing user should notice is these libraries depend on cldr-data. Does this clarify?
```
"dependencies": {
  "foo-number-format": "1.2.3", // Hypothetical foo-number-format depends on cldr-data.
  "bar-date-format": "4.5.6" // Hypothetical foo-number-format depends on cldr-data.
}
```

**Comment by johnnyreilly** [comment]
Not really, no.  Well, sort of...

Let me take you through it as I'm reading it.  

> If you're writing an end application that depends on i18n libraries, which in turn define their CLDR data dependency using bower's cldr-data, you need to:

> - List your i18n dependencies (obvious).
> - Setup a bower postinstall hook to populate the correct Unicode CLDR JSON data.

> On the bower.json of your application, list the i18n dependencies as you'd normally do. For example, let's say they are FooNumberFormat and BarDateFormat.

I think I understand what you're saying here but it wasn't obvious to me until I re-read it a number of times.  How about this:

-----

Let's imagine you're writing an end application that depends on some i18n libraries, which in turn define their CLDR data dependency using bower's cldr-data.  As an example, let's say your application depends on `foo-number-format` and `bar-number-format`.  To use these you simply list these dependencies in your application's `bower.json` like this:

```
"dependencies": {
  "foo-number-format": "1.2.3", // Hypothetical foo-number-format depends on cldr-data.
  "bar-date-format": "4.5.6" // Hypothetical bar-date-format depends on cldr-data.
}
```

You also need to setup a bower postinstall hook to populate the correct Unicode CLDR JSON data in a `.bowerrc` file.

**Comment by rxaviers** [comment]
Your suggestion seems fantastic to me. Would you be ok sending a PR with this change please?

**Comment by johnnyreilly** [comment]
Will do...

**Comment by goatandsheep** [comment]
Can I see an example of a bower postinstall hook? I've never done one before and I don't understand the purpose of the dependencies list. Say for example I'm using globalize, would I list globalize in my list of dependencies? And how do I ensure that the versions syncronize between the bowerrc and the bower?

**Comment by rxaviers** [comment]
This library is deprecated, please use npm cldr-data package instead.

**Comment by goatandsheep** [comment]
Can I install the `cldr-data-npm` through `bower install cldr-data` or must I use `bower install https://github.com/rxaviers/cldr-data-npm`? The [bower registry](https://bower.io/search/) is still showing this repo as the bower repo. The bower registry shows [twitter-cldr-npm](https://github.com/twitter/twitter-cldr-npm) as an alternative bower cldr source, but I don't know if it will work with globalize
